WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -1499,7 +1496,8 @@ translators: Block name. %s: The localized block name */
 /* A text for title label on Login epilogue screen */
 "Choose a site to open." = "اختيار موقع لفتحه.";
 
-/* Title of a Quick Start Tour */
+/* Title for the screen to pick a theme and homepage for a site.
+   Title of a Quick Start Tour */
```

Comment:
> I wonder if we shouldn't start creating a Peril rule...

I'm all up for this!

What would you think of also having SwiftLint custom rule as well? That would make the feedback loop to the devs faster. Peril could them be "the last bastion" of quality, a just-in-case check.

Reply:
The problem with a SwiftLint rule is it will catch all instances old and new in the whole codebase. And we have _a lot_ of existing instances of `NSLocalizedString` in the codebase already; it is not reasonable to require to change the key of all the _existing_ ones (and having them to be translated again from scratch in GlotPress because of that!), but it is reasonable to require new ones to follow the new convention, hence why Peril to flag only new one feels more fit for the job than SwiftLint.